### PR TITLE
change background color for demonstration 

### PR DESCRIPTION
**Pull request by y1c1**
:)

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 body {
-    background-color: #fcfcfc;
```
Comment (BethJ-20) [inline index.css:2]: hello I like your color choice!

Comment (BethJ-20) [inline index.css:2]: j

**Review by y1c1 — COMMENTED** [review]
looks good

**Review by BethJ-20 — COMMENTED** [review]
looks good!

**Pull request merged**